crates/bevy_gltf/src/loader.rs

```diff
@@ -612,14 +613,19 @@ fn load_node(
                     bottom: -ymag,
                     far: orthographic.zfar(),
                     near: orthographic.znear(),
+                    scaling_mode: ScalingMode::FixedHorizontal,
+                    scale: xmag,
```

Comment:
```suggestion
                    scale: xmag / 2.0,
```

doing the math from https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#orthographic-projection, it looks like this need to be divided by 2

Reply:
Amended the suggested changes